operator/src/main/resources/scripts/initializeDomainHomeOnPV.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/sh
+# Copyright (c) 2023, Oracle and/or its affiliates.
+
+scriptDir="$( cd "$(dirname "$0")" > /dev/null 2>&1 ; pwd -P )"
+if [ "${debug}" == "true" ]; then set -x; fi;
+
+. ${scriptDir}/utils_base.sh
+[ $? -ne 0 ] && echo "[SEVERE] Missing file ${scriptDir}/utils_base.sh" && exit 1
+UNKNOWN_SHELL=true
+
+checkEnv DOMAIN_HOME || exit 1
+
+ORIGIFS=$IFS
+trace "DOMAIN HOME is " $DOMAIN_HOME
+trace "Running the script as "`id`
+
+if [[ -f $DOMAIN_HOME ]]; then
+  if [[ -f $DOMAIN_HOME/config/config.xml ]] ; then
+    trace INFO "DOMAIN_HOME "$DOMAIN_HOME" already exists, no operation. Exiting with 0 return code"
+    exit 0
+  else
+    trace SEVERE "DOMAIN_HOME "$DOMAIN_HOME" is not empty and does not contain any WebLogic Domain. Please use an empty directory for domainHome"
```

Comment:
> Please use an empty directory for domainHome
Should this be something like "Please specify a domain home empty directory in 'domain.spec.domainHome' ." ?